libass/ass.c

```diff
@@ -612,11 +733,12 @@ static bool format_line_compare(const char *fmt1, const char *fmt2)
  * keep SBAS=1 default for custom format files.
  */
 static void custom_format_line_compatibility(ASS_Track *const track,
-                                             const char *const fmt,
-                                             const char *const std)
+                                             const ASS_FormatToken *fmt,
+                                             const ASS_FormatToken *std,
+                                             size_t num)
```

Comment:
This function isn't equivalent to its previous version: should be separate `n_fmt` and `n_std` instead of one `num`.

Reply:
I mean old version checked exact equality, but current version checks that one token list is a prefix of another. I.e. there's missing `n_fmt == n_std` check.